c_emulator/riscv_sim.c

```diff
@@ -217,7 +231,7 @@ char *process_args(int argc, char **argv)
   int c;
   uint64_t ram_size = 0;
   while(true) {
-    c = getopt_long(argc, argv,
+    c = getopt_long_only(argc, argv,
```

Comment:
Please update the commit message to reflect that this now works with a single-dash.

Reply:
Will do at the squash-and-merge stage.

Reply:
And this is *definitely* not yet resolved as a result, so do not mark it as such